src/App.jsx

```diff
@@ -14,33 +15,85 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+function getPrepareGoods(goods, { sortField, isReversed }) {
+  const prepareGoods = [...goods];
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  switch (sortField) {
+    case 'alphabetically':
+      prepareGoods.sort((a, b) => a.localeCompare(b));
+      break;
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+    case 'byLength':
+      prepareGoods.sort((a, b) => a.length - b.length);
+      break;
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+    default:
+      break;
+  }
+
+  if (isReversed) {
+    prepareGoods.reverse();
+  }
+
+  return prepareGoods;
+}
```

Comment:
The `getPrepareGoods` function currently defaults to doing nothing if the `sortField` is not recognized. Consider providing a default sorting behavior or logging a warning to help with debugging unexpected `sortField` values.